lesson3/hometask03_ankarut.rb

```diff
@@ -0,0 +1,54 @@
+require 'rspec'
```

Comment:
it redundant, `rspec/expectations` is enough. I will update your PR to add gemfile and delete `require 'rspec'`.